content/en/docs/reference/glossary/resource.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Resource
+id: resource
+date: 2025-02-09
+full_link: https://kubernetes.io/docs/reference/using-api/api-concepts/#standard-api-terminology
+short_description: >
+  A Kubernetes entity, representing a RESTful endpoint on the Kubernetes API server
+
+aka:
+tags:
+- fundamental
+---
+An entity in the Kubernetes type system, typically representing {{< glossary_tooltip text="objects" term_id="object" >}}.
+But sometime representing an operation on objects, such as a permission check.
```

Comment:
Is there somewhere "resource" is used like this? Seems odd to me to consider a permission check as a resource.

Reply:
From: https://kubernetes.io/docs/reference/using-api/api-concepts/#standard-api-terminology

> smaller number of API resource types are virtual in that they often represent operations on objects, rather than objects, such as a permission check 

I added this for completeness. It also isn't typically what I would think of as a "resource". But I did want to try and capture the definitions from existing material.

Reply:
It's apparently because we use "object" specifically when referring to concrete resource types like pods (i.e. resources that "exist" in the cluster), whereas something like a `CreateEviction`, while still a resource, wouldn't be an object

Reply:
So, a Pod is an object (it has spec, it has metadata). But the eviction subresource of a Pod is not an object. Both are (HTTP) resources - see https://datatracker.ietf.org/doc/html/rfc7231#page-7